### ISSUE TITLE
Decoupled service has no method .get()

### ISSUE THREAD

**Issue by sikko** [mgonto/restangular#1189]
Hi,

As per the documentation: https://github.com/mgonto/restangular#decoupled-restangular-service, a decoupled service should have a .get() method.

However, as you can see here: http://jsfiddle.net/sikko/8jq35w72/ the console throws an error: "[Service Name].get is not a function"

Strangely enough, when using the .all() notation it works: http://jsfiddle.net/sikko/o0qax2tz/1/

Is this a normal behaviour or a bug?

**Comment by gsjen** [comment]
+1

**Comment by mathielen** [comment]
+1

**Comment by digitlimit** [comment]
Having same issue.
Trying Restangular for the first time by been through a whole lot of errors, first issue of _.contains and now .get not working. Any help will be appreciated 

```
app.service('Company', ['Restangular', 
    function (Restangular)
    {
         return Restangular.service('company');
    }
]);
```

**Comment by bostrom** [comment]
Fixes as of 1.6.0.
